gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/util/UserAgent.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I don't know if it makes sense to have an entirely new class for this function, I didn't see an obvious place to include it though

Reply:
Potentially, you could just add it to the Channelizer or even Cluster classes if you make the static initializer changes. Leaving it here is fine too I think.